index.html

```diff
@@ -234,52 +268,50 @@ <h1 class="about_myself_heading">About me</h1>
       </ul>
     </section>
 
-    <section class="contact_section">
-      <h3>
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </h3>
-
-      <form
-        action="https://formspree.io/f/xgedowvw"
-        method="POST"
-        class="contact_form"
-      >
-        <input
-          class="name"
-          type="text"
-          placeholder="Full name"
-          required
-          maxlength="30"
-        />
-        <hr class="horizontal_line" />
-        <input
-          class="email"
-          type="email"
-          placeholder="Email address"
-          required
-        />
-        <hr class="horizontal_line" />
-        <textarea
-          class="message"
-          name="message"
-          id="message"
-          cols="30"
-          rows="10"
-          maxlength="500"
-          placeholder="Enter text here"
-          required
-        ></textarea>
-
-        <button class="submit_button" type="submit">Get in touch</button>
-      </form>
-    </section>
-
-    <hr class="horizontal_line horizontal_line_3" />
-
     <footer class="footer">
-      <div class="social-media social-media-footer">
-        <ul class="social-media-links">
+      <section class="contact_section" id="contact">
```

Comment:
- [ ] Good job correcting some of the fonts! 💯 , however, all over your project (mostly in the about me section) there are still discrepancies with the font styles shown on the Figma design.  

Look: 

![image](https://user-images.githubusercontent.com/57421823/149435349-5b1e775d-2b03-4ba9-a1a5-f52287314571.png)

As shown above, you just need to select the text in question, and look at the font properties on the right of the template.

Put special attention on the line-height, font-size, font-spacing, font-weight, font-family, letter-spacing.

Let's please kindly correct them so your project looks just like the Figma 💯